spec/features/roles_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'assigning roles' do
+  context 'with admin privilege' do
+    it 'assigns one role to user' do
+      admin = create :user, admin: true
+      login_as admin
```

Comment:
Sounds like if we leave the spec_helper method as is and use it everywhere we should be fine.